query/src/servers/mysql/mysql_interactive_worker.rs

```diff
@@ -232,16 +233,33 @@ impl<W: std::io::Write> InteractiveWorkerBase<W> {
         let client_ip = info.user_client_address.split(':').collect::<Vec<_>>()[0];
 
         let ctx = self.session.create_query_context().await?;
-        let user_manager = ctx.get_user_manager();
-        let user_info = user_manager
-            .get_user_with_client_ip(&ctx.get_tenant(), user_name, client_ip)
-            .await?;
-
-        let authed = user_info.auth_info.auth_mysql(&info.user_password, salt)?;
-        if authed {
-            self.session.set_current_user(user_info);
-        }
-        Ok(authed)
+        let auth_manager = ctx.get_auth_manager();
+        let credential = Credential::Password {
+            name: user_name.to_string(),
+            password: Some(salt.to_vec()),
```

Comment:
why assign salt to the password field?

Reply:
adding a stateless  https://github.com/datafuselabs/databend/pull/5600
you can start servers using ./scripts/ci/deploy/databend-query-standalone.sh and test with mysql client first

Reply:
`mysql_interactive_worker` seems to be using different auth algorithm from `auth_mgr` so I rollback the change here.